charts/datadog/templates/agent-priorityclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.agents.priorityClassCreate}}
+apiVersion: scheduling.k8s.io/v1
+description: Used for Datadog Agent Components to be scheduled with higher priority.
+kind: PriorityClass
+metadata:
+  name: {{ .Values.agents.priorityClassName | default (include "datadog.fullname" . ) }}
```

Comment:
the default name should contains the namespace since `PriorityClass` resources are cluster wide

Reply:
hum other cluster wild resource such as clusterrole use also "datadog.fullname". 

So lets consider it here ok